lib/puppet.rb

```diff
@@ -74,7 +74,7 @@ class << self
       end
 
       if locale_path
-        if Gem.loaded_specs['gettext-setup'].version < Gem::Version.new('0.8')
+        if GettextSetup.method(:initialize).parameters.count < 2
```

Comment:
I'm confused why this wouldn't be just `== 1`. It's called with one argument below. Perhaps a nit, but it just reads weird to have 2 here and 1 argument below.